### PR TITLE
Feature/handle fetch error

### PR DESCRIPTION
**Pull request by annamkummer**
# What (if any) features are you implementing?
- An modal for fetch errors
- A maximum of 20 countries to be used for each game, which reduces the number of coordinates the user will see that don't match their board
# What (if anything) did you refactor?
- Added a getRandomList function to utils.js to reduce some code in Game.
- Changed squiggle to bottom border in navlinks per instructor feedback
# Were there any issues that arose?
- n/a
# Is there anything that you need from your teammate?
- n/a
# Any other comments, questions, or concerns?
-

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -16,4 +16,23 @@ export const winningBoards = [
 
 export const getRandomIndex = (arr) => {
     return Math.floor(Math.random() * arr.length)
-  }
\ No newline at end of file
+  }
+
+export const getRandomList = (arr, length) => {
```
Comment (jkiernan12) [inline src/utils.js:21]: Nice refactor -- much DRYer!

**Review by jkiernan12 — APPROVED** [review]
This looks great -- nice job on this!

File: src/components/Game.js [hunk 2]
```diff
@@ -86,25 +84,22 @@ class Game extends Component {
       filteredCountries = this.state.allCountries
     }
 
+    let maxCountries = (filteredCountries.length < 20) ? filteredCountries.length : 20
```
Comment (NEwertKrocker) [inline src/components/Game.js:87]: I think you're right that this is gonna improve the user experience a LOT. It's no fun always hunting for fruitless coordinates!

File: src/components/Game.js [hunk 3]
```diff
@@ -219,13 +214,22 @@ class Game extends Component {
...
       .then(() => this.setFilteredCountries())
+      .catch(err => 
+        {
+          let message = (err.message === 'Failed to fetch') ? 'Oops! There was a problem. Please check your internet connection or try again later.' : err.message
+          this.setState({
+            error: message,
+            showModal: true
+          })
+        }
+      )
```
Comment (NEwertKrocker) [inline src/components/Game.js:225]: Nice! I will update the Cypress suite to make sure we're testing for this.

**Review by NEwertKrocker — APPROVED** [review]
Looks great!

**Pull request merged**